### PR TITLE
feat(ColorConversion): add a rgb to hex transformation method

### PR DESCRIPTION
**Pull request by tsanislavgatev**

### CODE REVIEW

File: packages/base/src/util/ColorConversion.js [hunk 1]
```diff
@@ -319,6 +319,28 @@ const HEXToRGB = hex => {
 	return rgbValues;
 };
 
+const RGBtoHEX = color => {
```
Comment (a-z-ivanov) [inline packages/base/src/util/ColorConversion.js:322]: color - either document it or destructure it like in HSLtoRGB.

File: packages/base/src/util/ColorConversion.js [hunk 2]
```diff
@@ -367,4 +389,6 @@ export {
 	HSLToRGB,
 	HEXToRGB,
 	RGBToHSL,
+	RGBToRGB,
```
Comment (a-z-ivanov) [inline packages/base/src/util/ColorConversion.js:392]: can be renamed to RGBStringToRGB, otherwise it makes no sense.

**Review by a-z-ivanov — CHANGES_REQUESTED** [review]

File: packages/base/src/util/ColorConversion.js (hunk 1 above)
Comment (tsanislavgatev) [reply packages/base/src/util/ColorConversion.js:326]: Documented it with the expected format

File: packages/base/src/util/ColorConversion.js (hunk 2 above)
Comment (tsanislavgatev) [reply packages/base/src/util/ColorConversion.js]: done

**Review by a-z-ivanov — APPROVED** [review]

**Pull request merged**